src/components/List/RowContent.js

```diff
@@ -348,50 +358,60 @@ class ListRowContent extends Component {
               </MobileOnlyColumn>
 
               <ContentColumn
-                key="primary"
+                key="primary-collapsed"
                 medium={6}
                 className={classnames({
                   column__content: true,
-                  "list-row--title": true,
+                  "list-row__title": true,
                   "column__content--expanded": isOpen,
                   "column__content--collapsed": !isOpen
                 })}
               >
                 <MultilineText
-                  responsiveSize={{ xSmall: "hecto", medium: "kilo" }}
+                  responsiveSize={staticProps.defaultResponsiveSize}
                 >
                   {title}
                 </MultilineText>
               </ContentColumn>
+
               {/* this class name is for automation purposes please do not remove or modify the name */}
               <ContentColumn
-                key="secondary"
+                key="secondary-collapsed"
                 medium={6}
-                className="column__content"
+                className={classnames({
+                  column__content: true,
+                  "list-row__title": true,
+                  "column__content--expanded": isOpen,
+                  "column__content--collapsed": !isOpen
+                })}
               >
                 <MultilineText
-                  key="collapsed"
-                  responsiveSize={{ xSmall: "hecto", medium: "kilo" }}
+                  responsiveSize={staticProps.defaultResponsiveSize}
                   className={classnames({
                     subtitle: true,
                     "list-row--subtitle": true,
-                    "subtitle--active": !isOpen,
-                    "subtitle--hidden": isOpen,
                     // this class name is for automation purposes please do not remove or modify the name
                     "subtitle--collapsed": true
                   })}
                 >
                   {subTitle}
                 </MultilineText>
+              </ContentColumn>
+
+              {/* this class name is for automation purposes please do not remove or modify the name */}
```

Comment:
Did you include this comment by mistake?

Reply:
Yes, I copypasted the <ContentColumn/> with this comment, so I'll fix that =_=